bin/logstash.lib.sh

```diff
@@ -81,6 +75,12 @@ setup_bundled_jdk_part() {
   fi
 }
 
+setup_classpath() {
+  for J in $(cd "${LOGSTASH_JARS}"; ls *.jar); do
+    CLASSPATH=${CLASSPATH}${CLASSPATH:+:}${LOGSTASH_JARS}/${J}
+  done
+}
+
```

Comment:
nit: this works, but I am adverse to shell functions that require specific environmental state (e.g. `CLASSPATH` to be unset, `LOGSTASH_JARS` to be set), and mutate other environmental state, because it is really easy to screw them up accidentally later.

```suggestion
setup_classpath() {
  local jar_directory="${1?jar directory required}"
  local classpath
  for J in $(cd "${jar_directory}"; ls *.jar); do
    classpath=${classpath}${classpath:+:}${jar_directory}/${J}
  done
  echo "${classpath}"
}
```

Which makes the later usage more straight-forward (e.g., no `unset CLASSPATH` required)

~~~ sh
  CLASSPATH="$(setup_classpath $LOGSTASH_JARS)"
~~~